docs/modules/usage/llms/custom-llm-configs.md

```diff
@@ -0,0 +1,109 @@
+# Custom LLM Configurations
+
```

Comment:
I didn't know this is possible. Is this automatic? I thought you always have to set the LLM in the UI to "switch".
Or is this for something else?

Reply:
Implemented [8 months ago](https://github.com/All-Hands-AI/OpenHands/pull/1552#issuecomment-2093998935) apparently. 🙊 In the discussion there Li Boxuan asked for it; the exact way to name these custom configs changed a bit in time.

This is how we define and switch [LLMs for evals](https://github.com/All-Hands-AI/OpenHands/tree/main/evaluation#evaluating-different-llms). Also when running headless, or in vscode debug configurations.

It has not been implemented for the UI. There's no way in the frontend to reference these custom configurations.